docs/installation/introduction.rst

```diff
@@ -315,36 +317,36 @@ on a single Node, these requirements add up.
-| *common to*    | Kubelet               |          |        |                                 |                             |
-| *any Node*     |                       |          |        |                                 |                             |
+| *requirements\ | Salt minion,          | 0.2 core | 0.5 GB | **40 GB root partition**        | 100 GB or more for /var     |
+| common to\     | Kubelet               |          |        |                                 |                             |
+| any Node*      |                       |          |        |                                 |                             |
 +----------------+-----------------------+----------+--------+---------------------------------+-----------------------------+
 
-These numbers are not accounting for highly unstable workloads or other sources
-of unpredictable load on the cluster services, and it is recommended to provide
-an additional 50% of resources as a safety margin.
+These numbers do not account for highly unstable workloads or other sources of
+unpredictable load on the cluster services. Providing a safety margin of an
+additional 50% of resources is recommended.
 
 Consider the `official recommendations for etcd sizing
-<https://github.com/etcd-io/etcd/blob/master/Documentation/op-guide/hardware.md>`_
-as the stability of a MetalK8s installation depends strongly on the backing
-``etcd`` stability (see :ref:`this note<Setup etcd partition>` for more
+<https://github.com/etcd-io/etcd/blob/master/Documentation/op-guide/hardware.md>`_,
+as the stability of a MetalK8s installation depends on the stability of the
+backing etcd (see the :ref:`etcd<Setup etcd partition>` section for more
 details). Prometheus and Alertmanager also require storage, as explained in
-:ref:`this section<Provision Prometheus Storage>`.
+:ref:`Provision Prometheus Storage`.
 
 .. _installation-intro-cloud:
 
 Deploying with Cloud Providers
 """"""""""""""""""""""""""""""
+
 When installing in a virtual environment, such as `AWS EC2`_ or `OpenStack`_,
-special care will be needed for adjusting networks configuration. Virtual
-environments often add a layer of security at the port level, which should be
-disabled, or circumvented with :ref:`IP-in-IP encapsulation<enable IP-in-IP>`.
+adjust network configurations carefully: virtual environments often add a layer
+of security at the port level, which must be disabled or circumvented with
+:ref:`IP-in-IP encapsulation<enable IP-in-IP>`.
 
 Also note that Kubernetes has numerous integrations with existing cloud
 providers to provide easier access to proprietary features, such as
-load balancers. For more information, see
-`this documentation article
-<https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/>`_.
+load balancers. For more information, review
+`this topic<https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/>`_.
```

Comment:
this does not work and prevents the build.
I would put:
```
review Kubernetes' topic on `Cloud Providers`_.

.. _Cloud Providers: https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/
```

Reply:
I stuck with "this topic" and fixed the bad link before I saw your review comment.

Reply:
just fixed the bad link again...